pkgs/tools/text/frece/update.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix-shell
```

Comment:
You can also use nix-update to automatically upgrade, without having to write a custom script.

Reply:
- https://github.com/Mic92/nix-update - This one right? 

Also, I just found out about https://github.com/ryantm/nixpkgs-update too.

I'm gonna take a look at both. Thanks for the suggestion. :smiley: